CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 0.5.0 / 2022-07-19
+
+- [ENHANCEMENT] gauge chart support additional units #481
+- [FEATURE] support echarts theme customization #480
+- [BUGFIX] line chart query error handling #479
+- [ENHANCEMENT] BE: support charts without datasource, add text plugin #478
+- [FEATURE] Add ability to show info icon next to panel header #465
+- [FEATURE] `thresholds` support in line chart #462
+- [FEATURE] echarts legends using `show_legend` in line chart #462
+- [ENHANCEMENT] enable data zoom by default in line chart #462
+- [ENHANCEMENT] add Bytes formatting to line and stats charts #462
+- [ENHANCEMENT] stat chart selectable text and theme improvements #462
+- [ENHANCEMENT] cue format improvements #463, #470
+- [ENHANCEMENT] Validator: support query types as plugins #459
+- [ENHANCEMENT] configurable charts location, remove configurability of base def #457
+- [ENHANCEMENT] align layout definition between frontend and backend #454
+- [ENHANCEMENT] release process and license header improvements #452, #453, #456
```

Comment:
#453 is actually a different enhancement. It is adding a field `version` to every resources like `dashboard` or `datasource`. It is a number increased during each update of the resource.

```
[ENHANCEMENT] add a version number on every resources managed. This number is increased when a resource is updated. #453
```